### PR TITLE
EP-37 Releng - corundum added

### PR DESCRIPTION
**Pull request by arjantijms**
EP-37 Releng - corundum added

### CODE REVIEW

**Comment by arjantijms** [comment]
> Is the ant-contrib jar supposed to be added?

Yes, it indeed is. In ant-based projects this is done more often, and in the original code this was done too.

File: releng/corundum-src/corundum.properties [hunk 1]
```diff
@@ -0,0 +1,303 @@
+# The name of the product for display purposes
+
+product.name = Product
```
Comment (Pandrex247) [inline releng/corundum-src/corundum.properties]: Are these line endings going to be an issue?

File: releng/corundum-src/extensions/eclipse.org/sign-jar.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<project name="corundum.eclipse.sign-jar">
+
+	<macrodef name="sign-jar">
+		<attribute name="location" />
+		<sequential>
```
Comment (Pandrex247) [inline releng/corundum-src/extensions/eclipse.org/sign-jar.xml]: Spaces > tabs

File: releng/corundum-src/corundum.properties (hunk 1 above)
Comment (arjantijms) [reply releng/corundum-src/corundum.properties]: Unlikely, have been in there forever

File: releng/corundum-src/macros/build-configuration.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<!-- 
+ ******************************************************************************
+ * Copyright (c) 2016 Oracle
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
```
Comment (Pandrex247) [inline releng/corundum-src/macros/build-configuration.xml]: Presumably it's fine for us to be copy-pasting these?

Comment (arjantijms) [reply releng/corundum-src/macros/build-configuration.xml]: It's copy/paste with adhering to the license, which is also EPL.

File: releng/corundum-src/extensions/eclipse.org/sign-jar.xml (hunk 2 above)
Comment (arjantijms) [reply releng/corundum-src/extensions/eclipse.org/sign-jar.xml]: Good one - tabs > spaces applied

**Pull request merged**